File: codingbat/python/src/warmup-2/string_match.py
def string_match(a, b):
  small = len(a) if len(a) < len(b) else len(b);
  count = 0;
  i = 0;
  while i < small-1:
    if a[i:i+2] == b[i:i+2]:
      count += 1;
    i += 1;
  return count;

File: codingbat/java/src/warmup-2/stringMatch.java
public int stringMatch(String a, String b) {
  int count = 0;
  for(int i=0; i<(a.length() < b.length() ? a.length() : b.length())-1; i++){
    if(a.substring(i,i+2).equals(b.substring(i,i+2))){
      count++;
    }
  }
  return count;
}
